### ISSUE TITLE
requires_grad for all_imgs and all_imgs_tf in cluster_greyscale_twohead.py

### ISSUE THREAD

**Issue by tareshsethi** [xu-ji/IIC#5]
Why is it required for these two tensors to be set with requires_grad for clustering yet not for segmentation? More so do not understand why one would set requires_grad to True for input images into a conv net, as I expect they should remain static. Thanks in advance!

**Comment by xu-ji** [comment]
Hi, it doesn't matter whether this is set or not, training is the same. When this is set, extra gradients are computed at the input images, but we never use these gradients in any way (the optimiser only uses model parameter gradients because it only updates model parameters). Thanks for pointing out it's redundant, I've removed it.
